src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--31">
+      <div class="calendar__day calendar__day--1 start-day-mon"></div>
```

Comment:
The class 'start-day-mon' suggests that the calendar will always start on Monday. Instead, consider making the start day dynamic based on the month and year.

Reply:
How can I do this without JS???? I thought that would be enough added a class....